libs/community/langchain_community/vectorstores/yellowbrick.py

```diff
@@ -24,19 +27,26 @@
 
 class Yellowbrick(VectorStore):
     """Yellowbrick as a vector database.
-
     Example:
         .. code-block:: python
             from langchain_community.vectorstores import Yellowbrick
             from langchain_community.embeddings.openai import OpenAIEmbeddings
             ...
     """
 
+    LSH_INDEX_TABLE: str = "_lsh_index"
+    LSH_HYPERPLANE_TABLE: str = "_lsh_hyperplane"
+    CONTENT_TABLE: str = "_content"
+
     def __init__(
         self,
         embedding: Embeddings,
         connection_string: str,
```

Comment:
(not required for PR) init should be expanded beyond a connection string, so folks can pass either an existing connection or a connection pool

Reply:
Yep, added this. Thanks!